### PR TITLE
Fix share sample duplicate names

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Fixed 403 error being displayed when attempting to share samples when the target project had samples with the same names. Updated to list the samples which will not be shared due to either the identifiers or the names being the same in the source and target project.

shareduplicates

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [x] Tests added (or description of how to test) for any new features.
* [x] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/main/resources/i18n/messages.properties [hunk 1]
```diff
@@ -371,7 +371,8 @@ ShareSamples.remove=Remove
 ShareSamplesList.title=Review samples to share
 ShareSamples.no-samples.message=All samples exist in the target project
 ShareSamples.no-samples.description=Since these samples exists, there is no reason to re-share them.
-ShareSamples.some-samples.message={0} samples exist in the target project and will not be re-shared.
+ShareSamples.some-samples-same-ids.message=The following samples will not be shared as they exist in the target project with the same identifiers:
+ShareSamples.some-samples-same-names.message=The following samples will not be shared as they exist in the target project with the same names but different identifiers:
```
Comment (ericenns) [inline src/main/resources/i18n/messages.properties:375]: Can you change this translation to the following:
`The following samples cannot be shared/moved, as the target project already contains samples with the same names.`

File: src/main/resources/i18n/messages.properties [hunk 2]
```diff
@@ -371,7 +371,8 @@ ShareSamples.remove=Remove
 ShareSamplesList.title=Review samples to share
 ShareSamples.no-samples.message=All samples exist in the target project
 ShareSamples.no-samples.description=Since these samples exists, there is no reason to re-share them.
-ShareSamples.some-samples.message={0} samples exist in the target project and will not be re-shared.
+ShareSamples.some-samples-same-ids.message=The following samples will not be shared as they exist in the target project with the same identifiers:
```
Comment (ericenns) [inline src/main/resources/i18n/messages.properties:374]: Can you change this translation to the following:
`The following samples cannot be shared/moved, as the target project already contains them.`

**Review by ericenns — CHANGES_REQUESTED** [review]
Just a few translation changes, otherwise this looks good.

File: src/main/resources/i18n/messages.properties (hunk 1 above)
Comment (deepsidhu85) [reply src/main/resources/i18n/messages.properties]: Updated in 360cd24d260ee677c22e2f05e2539eb84949d713

**Review by ericenns — APPROVED** [review]
👍 This looks good to me, thanks for fixing this!

**Pull request merged**